packages/create-cloudflare/src/common.ts

```diff
@@ -44,27 +47,31 @@ export const validateProjectDirectory = (relativePath: string) => {
 	}
 
 	// Ensure the name is valid per the pages schema
-	const projectName = basename(path);
-	const invalidChars = /[^a-z0-9-]/;
-	const invalidStartEnd = /^-|-$/;
-
-	if (projectName.match(invalidStartEnd)) {
-		return `Project name cannot start or end with a dash.`;
-	}
+	// Skip this if we're initializing from an existing workers script, since some
+	// previously created workers may have names containing capital letters
+	if (!args.existingScript) {
+		const projectName = basename(path);
+		const invalidChars = /[^a-z0-9-]/;
+		const invalidStartEnd = /^-|-$/;
+
+		if (projectName.match(invalidStartEnd)) {
+			return `Project name cannot start or end with a dash.`;
```

Comment:
Nit: I just noticed that the last error message says "Project name**s** must ..." (plural) whilst the first two say "Project name cannot/must ..." (singular)

Could you align them? 🙏  (I think the plural version is the correct one)

Reply:
good call 👍